kitchen-tests/.kitchen.travis.yml

```diff
@@ -2,8 +2,6 @@
 driver:
   name: dokken
   privileged: true
-  chef_image: chef/chef
-  chef_version: current
```

Comment:
i wonder if something like this was bizzarely necessary magic and now we're somehow getting cached ruby-2.5.0 versions of chef-client in the base image?  (which seems backwards, but maybe these flags have the effect of disabling the precached images and then later we do the kitchen-appbundle-updater dance to install what we actually need?).